tuist/tuist
# Support for embedding a CLI app within a macOS application

Resolves https://github.com/tuist/tuist/issues/4205

### Short description 📝

The issue is so close to #4973, but extended for embedding CLI apps.

### How to test the changes locally 🧐

1. Checkout a sample project:
```
https://github.com/serejahh/tuist/tree/feature/enable-embedding-CLI-app-within-mac-application/projects/tuist/fixtures/macos_app_with_app_inside
```
2. Run `tuist generate`
3. Make sure the project is generated

### Contributor checklist ✅


### Reviewer checklist ✅
